.github/workflows/autoformat.yml

```diff
@@ -183,7 +183,7 @@ jobs:
           git merge base/${{ needs.check-comment.outputs.base_ref }}
 
       - name: Download patch
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
The artifact does exist:

https://github.com/mlflow/mlflow/actions/runs/7840927213/job/21396379870

<img width="925" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/2ddac614-fe47-4f2f-969f-2e0d48e57cba">

but `actions/download-artifact@v3` failed to find it.

Reply:
https://github.com/actions/download-artifact?tab=readme-ov-file#breaking-changes says

> Downloading artifacts that were created from action/upload-artifact@v3 and below are not supported.